packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -440,7 +440,15 @@ export default class TabAdvanced extends Component {
                         value={migrationNetwork}
                         xoType='network'
                       >
-                        {migrationNetwork !== undefined ? <Network id={migrationNetwork.id} /> : _('noValue')}
+                        {pool.otherConfig['xo:migrationNetwork'] === undefined ? (
+                          _('noValue')
+                        ) : migrationNetwork !== undefined ? (
+                          <Network id={migrationNetwork.id} />
+                        ) : (
+                          <span className='text-danger'>
+                            {_('networkIdDeleted', { networkID: pool.otherConfig['xo:migrationNetwork'] })}
+                          </span>
```

Comment:
It is a choice I made, in my opinion it was clearer to add a message

Reply:
I don't know if it's a good idea to override the default behavior, since users are used to it.
@pdonias, what do you think?

Reply:
I guess it can't hurt to have a more explicit error message. However:
- the user should be able to easily copy the ID (`Copiable`)
- we shouldn't assume what happened: even though the network has most likely been deleted, all we know for sure is that we just can't find a network with that ID. We don't know if it has ever existed or if it has been deleted.